taxonomies/ingredients.txt

```diff
@@ -14555,7 +14555,7 @@ carbon_footprint_fr_foodges_value:fr:4.2
 <en:cream
 en:pasteurized cream
 de:pasteurisierte Sahne
-fr:crème pasteurisé
+fr:crème pasteurisé, crème pasteurisée
```

Comment:
```suggestion
fr:crème pasteurisée
```

"crème pasteurisé" is incorrect

Reply:
I wonder if there are entries with just é.

Reply:
@aleene : there may be badly spelled entries, but we should not add them as synonyms, as we will use the taxonomy to do things like spell correction etc.